packages/gatsby-image/src/__tests__/__snapshots__/index.js.snap

```diff
@@ -54,19 +56,157 @@ exports[`<Image /> should render fluid images 1`] = `
       style="background-color: lightgray; position: absolute; top: 0px; bottom: 0px; opacity: 1; right: 0px; left: 0px; transition-delay: 0.5s;"
       title="Title for the image"
     />
-    <img
-      alt=""
-      class="placeholder"
-      src="string_of_base64"
-      style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: center; opacity: 1; transition-delay: 0.5s; color: red;"
+    <picture>
```

Comment:
I think the diff from here to line 91 is a bit misleading; all that changed for the existing snapshots was the addition of a `<picture>` tag around the base64 placeholder.